src/Apps/Artwork/Components/ArtworkImageBrowser/ArtworkActionsSaveButton.tsx

```diff
@@ -4,40 +4,86 @@ import {
   BellIcon,
   HeartFillIcon,
   HeartIcon,
+  Popover,
+  Text,
 } from "@artsy/palette"
 import * as React from "react"
 import { createFragmentContainer, graphql } from "react-relay"
 import { useSaveArtwork } from "Components/Artwork/SaveButton/useSaveArtwork"
 import { ArtworkActionsSaveButton_artwork$data } from "__generated__/ArtworkActionsSaveButton_artwork.graphql"
 import { UtilButton } from "./UtilButton"
+import { ArtworkAuctionRegistrationPanelFragmentContainer } from "Apps/Artwork/Components/ArtworkImageBrowser/ArtworkAuctionRegistrationPanel"
+import { useSystemContext } from "System"
 
 interface ArtworkActionsSaveButtonProps {
   artwork: ArtworkActionsSaveButton_artwork$data
 }
 const ArtworkActionsSaveButton: React.FC<ArtworkActionsSaveButtonProps> = ({
   artwork,
 }) => {
+  const { isLoggedIn } = useSystemContext()
   const { handleSave } = useSaveArtwork({
     isSaved: !!artwork.is_saved,
     artwork,
     contextModule: ContextModule.artworkImage,
   })
 
-  const isOpenSale = artwork.sale?.isAuction && !artwork.sale?.isClosed
-
+  const {
+    isAuction,
+    isClosed,
+    isLiveOpen,
+    isRegistrationClosed,
+    registrationStatus,
+    liveStartAt,
+  } = artwork.sale ?? {}
+  const isOpenSale = isAuction && !isClosed
   const isSaved = !!artwork.is_saved
+  const registrationAttempted = !!registrationStatus
+  const ignoreAuctionRegistrationPopover =
+    !isLoggedIn ||
+    !liveStartAt ||
+    isSaved ||
+    isRegistrationClosed ||
+    isLiveOpen ||
+    registrationAttempted
 
   // If an Auction, use Bell (for notifications); if a standard artwork use Heart
   if (isOpenSale) {
     const FilledIcon = () => <BellFillIcon fill="blue100" />
 
     return (
-      <UtilButton
-        name="bell"
-        Icon={isSaved ? FilledIcon : BellIcon}
-        label="Watch lot"
-        onClick={handleSave}
-      />
+      <Popover
+        title={
+          <Text variant="sm-display">
+            Register ahead of time to bid in this auction and get notifications
```

Comment:
**suggestion**: We might consider adding this copy to our [translations file](https://github.com/artsy/force/blob/fab739545f31ae264d6c660aed252e0e15822188/src/System/i18n/locales/en-US/translation.json).

Reply:
I think we also discussed a slight copy change to remove "ahead of time" when the auction does not have a specific registration deadline.

Registration deadline: Register ahead of time to bid in this auction and get notifications for this lot.

No registration deadline: Register to bid in this auction and get notifications for this lot.

Reply:
[Added changes](https://github.com/artsy/force/pull/11171/commits/ef77693296d52ac794136916097252eb1ce15472) and also updated review app